README.adoc

```diff
@@ -34,7 +34,7 @@ I've tried to add the rationale behind the rules (if it's omitted I've assumed i
 
 I didn't come up with all the rules out of nowhere - they are mostly based on my extensive career as a professional software engineer, feedback and suggestions from members of the Rails community and various highly regarded Rails programming resources.
 
-NOTE: Some of the advice here is applicable only to Rails 4.0+.
+NOTE: Some of the advice here is applicable only to the latest version of Rails.
```

Comment:
Maybe "Most of the advice here is applicable only to the recent versions of Rails"?

Reply:
👍 Updated.